docs/data/data-grid/filtering/filtering.md

```diff
@@ -309,14 +309,68 @@ The example below demonstrates how to achieve server-side filtering.
 
 ## Quick filter
 
-The grid does not natively include quick filtering.
-However, it can be implemented as in the demo below.
+The package also provides a component `<GridToolbarQuickFilter/>` which filters rows without specifying the concerned columns.
+By default, the quick filter considers the input as a list of values separated by space and keeps only rows that contain all the values.
 
 {{"demo": "QuickFilteringGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-> ⚠️ This feature isn't natively implemented in the grid package. It's coming.
->
-> 👍 Upvote [issue #2842](https://github.com/mui/mui-x/issues/2842) if you want to see it land faster.
+### Modifying the logic
+
+The logic quick filter logic can be switched to filter rows that contain _at least_ one value instead for each of them.
+To do so, set `quickFilterLogic` equals to `GridQuickFilterLogic.Or` as follow:
+
+```js
+initialState={{
+  filterModel: {
+    quickFilterLogic: GridQuickFilterLogic.Or
+  }
+}}
+```
+
+The quick filter will only consider columns with types `string` or `number`
```

Comment:
Could it support `singleSelect` ?

And below you talk about a `date` column.
Are `string` and `number` the only supported by default ? If so, maybe this sentence should be reworded.

Reply:
Yes, single-select could be supported. I wanted first to validate the global solution before doing this particular case. About the date, there is quite a lot of way to specify a date in a text area